src/us/kbase/narrativejobservice/NarrativeJobServiceServer.java

```diff
@@ -94,107 +87,113 @@
     public static final String CFG_PROP_AWE_READONLY_ADMIN_USER = "awe.readonly.admin.user";
     public static final String CFG_PROP_AWE_READONLY_ADMIN_PWD = "awe.readonly.admin.pwd";
     public static final String CFG_PROP_AWE_READONLY_ADMIN_TOKEN = "awe.readonly.admin.token";
+
     public static final String CFG_PROP_MONGO_HOSTS = "mongodb-host";
     public static final String CFG_PROP_MONGO_DBNAME = "mongodb-database";
     public static final String CFG_PROP_MONGO_USER = "mongodb-user";
     public static final String CFG_PROP_MONGO_PWD = "mongodb-pwd";
+    public static final String CFG_PROP_MONGO_HOSTS_UJS = "ujs-mongodb-host";
```

Comment:
ONLY IMPORTANT PART HERE, ADDED MORE VARIABLES for MONGODB, REST IS AUTOFORMATTING